### PR TITLE
LDAP mappings for superset

### PR DESCRIPTION
**Pull request by maulikjs**

### CODE REVIEW

File: superset/templates/odh_cr.yaml [hunk 1]
```diff
@@ -104,7 +104,21 @@ spec:
...
+      'cn=mhicks-all,ou=adhoc,ou=managedGroups,dc=redhat,dc=com': ['CCX'],
+      'cn=candlepin,ou=adhoc,ou=managedGroups,dc=redhat,dc=com': ['Subscriptions'],
+      'cn=cee-pel,ou=adhoc,ou=managedGroups,dc=redhat,dc=com': ['Subscriptions']
+      }
+
+      # the LDAP user attribute which has their role DNs AUTH_LDAP_GROUP_FIELD =
+      AUTH_LDAP_GROUP_FIELD= 'memberOf'
+
+      # if we should replace ALL the user's roles each login, or only on
+      registration AUTH_ROLES_SYNC_AT_LOGIN = False
```
Comment (rimolive) [inline superset/templates/odh_cr.yaml]: This doesn't seem to work the way it is. Did you test if this line will work?

Comment (accorvin) [reply superset/templates/odh_cr.yaml]: I think we want this value to be true.

Comment (maulikjs) [reply superset/templates/odh_cr.yaml]: ah yup, I forgot to change this from my dev environment 🤕

Comment (rimolive) [reply superset/templates/odh_cr.yaml]: I meant the `registration` word at the beginning of this line. It seems like it's part of the code, not the comment.

**Review by anishasthana — APPROVED** [review]
/lgtm

**Pull request merged**